core/src/main/java/de/raphaelebner/roomdatabasebackup/core/RoomBackup.kt

```diff
@@ -369,7 +368,11 @@ class RoomBackup(var context: Context) : FragmentActivity() {
             }
             BACKUP_FILE_LOCATION_CUSTOM_DIALOG -> {
                 backupFilename = filename
-                permissionRequestLauncher.launch(arrayOf(WRITE_EXTERNAL_STORAGE, READ_EXTERNAL_STORAGE))
+                if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
+                    permissionRequestLauncher.launch(arrayOf(READ_MEDIA_IMAGES, READ_MEDIA_AUDIO, READ_MEDIA_VIDEO))
+                } else {
+                    permissionRequestLauncher.launch(arrayOf(WRITE_EXTERNAL_STORAGE, READ_EXTERNAL_STORAGE))
```

Comment:
Since you removed `WRITE_EXTERNAL_STORAGE` from the manifest, we need to remove it here and on line 506 as well.